in/unit/ptc-vault.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=start vault
+After=network.target
+
+[Service]
+ExecStart=/usr/bin/vault agent -config=/home/unix/ptc/vault-agent/vault-agent.hcl
```

Comment:
use `%h` for $HOME as it's more portable.
https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Specifiers

Reply:
`In case of the system manager this resolves to "/root". Note that this setting is not influenced by the User= setting configurable in the [Service] section of the service unit`

Would this not resolve to `/root` instead of the `/home/unix/ptc` path if that statement is correct from the doc? 
The request here is to change 
`-config=/home/unix/ptc/vault-agent/vault-agent.hcl`
to
`-config=%h/ptc/vault-agent/vault-agent.hcl`

yes?

Reply:
Oh that's a very good point - I'm not exactly sure about this as it was initially meant to run as a user-level daemon.
Please ignore!